rockspec/apisix-master-0.rockspec

```diff
@@ -48,6 +48,7 @@ dependencies = {
     "lua-tinyyaml = 0.1",
     "iresty-nginx-lua-prometheus = 0.20190917",
     "jsonschema = 0.4",
+    "lua-resty-ipmatcher = 0.3",
```

Comment:
We can remove this line, and update `lua-resty-radixtree` to `1.6-1`:

```diff
--- a/rockspec/apisix-master-0.rockspec
+++ b/rockspec/apisix-master-0.rockspec
@@ -41,14 +41,13 @@ dependencies = {
     "lua-resty-cookie = 0.1.0",
     "lua-resty-session = 2.24",
     "opentracing-openresty = 0.1",
-    "lua-resty-radixtree = 1.5",
+    "lua-resty-radixtree = 1.6-1",
     "lua-protobuf = 0.3.1",
     "lua-resty-openidc = 1.7.2-1",
     "luafilesystem = 1.7.0-2",
     "lua-tinyyaml = 0.1",
     "iresty-nginx-lua-prometheus = 0.20190917",
     "jsonschema = 0.4",
-    "lua-resty-ipmatcher = 0.3",
 }
```

Reply:
@membphis 
I think it would be better to list lua-resty-ipmatcher as a dependency here since APISIX depends on it directly. If one day we remove lua-resty-radixtree, lua-resty-ipmatcher should not be removed.

Reply:
You are right, we can fix this later.